### DISCUSSION TITLE
How to bulk insert with SQLite

### DISCUSSION THREAD

**Discussion by Mwni** [prisma/prisma#12787] [Q&A]
The current SQLite implementation does not allow for more than 500-1000 `create` commands per second, even when wrapped into a transaction. SQLite in theory would be capable of about 100k per second.

Now the solution would be to use `createMany`, but that's not supported.

**Comment by austincrim** [comment]
Hey @Mwni 👋 ,

The current workaround is to use the [`$executeRaw`](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#executeraw) method on the Prisma Client. You can find more discussion in [this existing issue](https://github.com/prisma/prisma/issues/10710). 

Feel free to add your experience and +1 to help us prioritize!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
